Misc/NEWS.d/next/Security/2023-01-24-16-12-00.gh-issue-101283.9tqu39.rst

```diff
@@ -0,0 +1,6 @@
+Windows version of :meth:`subprocess.Popen.__init__` with
+``shell=True`` made its shell search algorithm protected from dropping
+a program named ``cmd.exe`` into a current directory. Now the current
+directory is used as the last resort if system environment variables
```

Comment:
It might be good to clarify this as "... if system environment variables from :data:`os.environ` are ..." to disambiguate from anyone thinking that the Popen `env=` parameter is used.

Reply:
I've simplified the wording as @zooba advised so it isn't applicable anymore.